### ISSUE TITLE
Cannot edit any material properties in new inherited scene

### ISSUE THREAD

**Issue by mamin** [godotengine/godot#72942]
### Godot version

4.0 RC 1

### System information

Linux 

### Issue description

Cannot edit material properties in new inherited scene in Godot 4 RC 1.
I expect to be able to edit the material properties. Using Godot 3.5.1 this works.

### Steps to reproduce

Export a glb file from blender.
Import into Godot 4.0.
Create new inherited scene.
Try editing roughness or any material property in the new inherited scene, you will find none of them can be edited.

### Minimal reproduction project

[GlbNotEditable.zip](https://github.com/godotengine/godot/files/10693364/GlbNotEditable.zip)

**Comment by Flavelius** [comment]
Not being able to edit the embedded material is not the issue (it's an issue that this is possible in 3.5.1), as the material is not an editable instance, it's part of the imported file.
The workaround is to open the advanced importer and from theactions menu at the top chose 'Extract Materials'. This creates editable copies.
But be aware that there are other usability issues (https://github.com/godotengine/godot/issues/72585) or bugs (https://github.com/godotengine/godot/issues/72587) that currently interfere with this _normal_ workflow aswell though.

**Comment by mamin** [comment]
I disagree. The Godot editor itself says "create new inherited scene to edit this scene" or something like that.
But then upon creating a new inherited scene it is actually not editable.
What was the whole point of creating a new inherited scene if it is not going to be edited.

In 3.5.1 the scene materials are editable upon creating a new inherited scene. That is NOT a bug in 3.5.1 as you are suggesting. That's the whole point of creating a new inherited scene and that's what the Godot editor itself prompts the user to do to be able to edit the scene. Godot 3.5.1 delivers on that promise, Godot 4 doesn't.

I think Godot 4 should reproduce the behavior in 3.5.1. 
For users migrating to Godot 4, they expect behavior parity with 3.5.1.
By default Godot 4 should do whatever is necessary to make the materials editable.
it shouldn't require the user to take extra steps to achieve feature parity with 3.5.1. Godot 4 should make things easier, not make them harder by adding more manual steps to achieve what worked automatically in 3.5.1.

Regardless of if this is "intentional" or not in Godot 4, for users migrating from 3.5.1 it is unexpected and people will be left wondering why isn't this working.

**Comment by mamin** [comment]
> Not being able to edit the embedded material is not the issue (it's an issue in 3.5.1 if that this is possible there), as the material is not an editable instance, it's part of the imported file. The workaround/workflow is to open the advanced importer and from the 'Actions..' menu at the top chose 'Extract Materials'. This creates editable copies. But be aware that there are other usability issues (#72585) or bugs (#72587) that currently interfere with this _normal_ workflow aswell though.

I followed the instructions provided:

1. From Actions select "Extract Materials"
2. Created a new inherited scene after re-importing.

In the new inherited scene I still cannot edit the any of the materials.

**Comment by Flavelius** [comment]
I just tried this in 3.5.1 and it works as technically expected (although as you said not desired for users), changes to materials inside meshes is not persisted inside inherited scenes.
Maybe the editor wrongly gave you the impression that it works because in 3.x meshes are imported as separate assets by default (atleast for me) in which case it will work (as they are separate assets and the changes are saved to those mesh resources). The same will and does work in 4.0 (just tested with rc1).
There menu name is just unfortunately named 'Set Mesh save Paths', where it's actually the same underlying functionality as 'Extract Materials'. It will extract the meshes to external resources which allows you to persist changes to their materials aswell, _and even allow edits in inherited scenes_.

**Comment by mamin** [comment]
By selecting "Set Mesh save paths", I am now able to edit the materials in the new inherited scene. Thank you for your help with that.

I'm just a user of Godot, so if it's a misbehavior or not in Godot 3.5, I came to rely on it. As a user, I don't really care about the internal details and what's right or wrong in technical sense. I care about usability.

And in the usability sense, Godot 4 has made this more complicated. 
How will users know they need to do these extra steps?
This is a fundamental change in the user experience when importing assets.

I think it should be the default behavior otherwise many people will go to forums/discord or even end up filing a bug report to github like did.

I asked for help on this in the Godot discord, and I also searched the internet for solution to this problem regarding Godot4 and it seems like there are no good answers and other people do have this problem.

There is a huge communication gap about this change and it's a blocker really cause editing material properties in Godot after importing is almost always required to get good looking results.

**Comment by clarence112** [comment]
there's a weird extra side effect of this bug:
if you attach a script with an @export var to the scene's root node, you still can't edit the material after instantiating it

<img width="442" height="90" alt="Image" src="https://github.com/user-attachments/assets/28893cbf-6642-4e9d-ab91-dbb9f0ab1dcb" />

<img width="275" height="90" alt="Image" src="https://github.com/user-attachments/assets/24f8934b-5e1d-4986-80cf-914d85ba5c1a" />

<img width="275" height="552" alt="Image" src="https://github.com/user-attachments/assets/554eb5de-b1fa-4852-9fdc-b94b500e083b" />

**Comment by Christakou** [comment]
Just reporting this bug still exists in 4.6
